### ISSUE TITLE
FAB & Button onPress not working with BottomSheet component

### ISSUE THREAD

**Issue by adilusman51** [callstack/react-native-paper#2767]
### Current behaviour

I have placed FAB inside a parent view with absolute position. When I click the FAB, ripple effect is visible but onPress event is not working. It is not working on Android with Expo Go Client and Standalone Build.
It is working properly in Expo Go Client on iOS simulator

### Expected behaviour

onPress event should be triggered when FAB is pressed

### Code sample

```
<View
	style={{
	flex: 1,
	paddingHorizontal: margin,
	paddingTop: 16,
	backgroundColor: theme.colors.surface,
	}}
	>
		<PaperFAB
			icon='plus'
			style={{
				position: 'absolute',
				right: 24,
				top: -20,
				elevation: theme.elevation,
				}}
			onPress={() => console.log('FAB is pressed')}
		/>
</View>
```

### Screenshots (if applicable)

![WhatsApp Image 2021-06-09 at 8 40 34 AM](https://user-images.githubusercontent.com/26340587/121289638-bc388080-c8fe-11eb-9cc3-3ec33b7bbf9e.jpeg)

### Your Environment

| software              | version
| --------------------- | -------
| android        | 29
| react-native          | 0.63
| react-native-paper    | 4.9.1
| node                  | 12.19.0
| yarn           | 1.22.5
| expo sdk              | 41.0.1

**Comment by adilusman51** [comment]
This issue is caused when Bottom Sheet from [reanimated-bottom-sheet](https://github.com/osdnk/react-native-reanimated-bottom-sheet) which is recommended library in the  [react native paper documentation](https://callstack.github.io/react-native-paper/recommended-libraries.html). 

FAB, Button, Checkbox and other which require focus/touch event do not work with `BottomSheet` from above mentioned package

**Comment by lukewalczak** [comment]
@adilusman51 looks like the issue is related to the bottom sheet library, and with its repository there is an issue about not working `TouchableOpacity` rendered within `renderContent` along with proposed [solution](https://github.com/osdnk/react-native-reanimated-bottom-sheet/issues/16#issuecomment-530017990). Did you try that?

**Comment by adilusman51** [comment]
@lukewalczak Thanks for your help, I did not know about above mentioned solution, thanks for pointing me in this direction. This issue was in fact due the bottom sheet library implementation, and is not due to react-native-paper.

I ended up switching to a different library for my use case [gorhom/react-native-bottom-sheet](https://github.com/gorhom/react-native-bottom-sheet), this library use reanimated v2. Former was implemented using reanimated ver1.

@lukewalczak Once again thanks a lot for your help. I am closing this issue for now.
